app/build.yaml

```diff
@@ -0,0 +1,19 @@
+# Read about `build.yaml` at https://pub.dartlang.org/packages/build_config
+# To update generated code, run `pub run build_runner build`
+targets:
+  pub_dartlang_org:
+    sources:
+      - 'lib/shared/*.dart'
+      - 'lib/search/backend*.dart'
+    builders:
+      json_serializable:
+        options:
+          require_library_directive: false
+          header: |+
+           // Copyright (c) 2017, the Dart project authors. Please see the AUTHORS file
```

Comment:
2018?

Reply:
I left it like this to keep the other files unchanged – when we change something else, we can bump those...